lib/senders/translator.js

```diff
@@ -47,7 +47,15 @@ module.exports = function (locales, defaultLanguage) {
       var supportedLanguages = []
       for (var i = 0; i < translations.length; i++) {
         var t = translations[i]
-        var language = i18n.normalizeLanguage(i18n.languageFrom(t.locale_data.messages[''].lang))
+        const localeMessageData = t.locale_data.messages['']
+
+        if (localeMessageData.lang === 'ar') {
+          // NOTE: there seems to be some incompatibility with Jed and Arabic plural forms from Pontoon
```

Comment:
I spent too much time trying to find the incompatibility and it seems like this fix should suffice in the meantime